WalletWasabi/BitcoinCore/Configuration/Whitening/WhiteEntry.cs

```diff
@@ -10,7 +11,7 @@ public abstract class WhiteEntry
 		public string Permissions { get; private set; } = "";
 		public EndPoint? EndPoint { get; private set; } = null;
 
-		public static bool TryParse<T>(string value, Network network, out T whiteEntry) where T : WhiteEntry, new()
+		public static bool TryParse<T>(string value, Network network, [NotNullWhen(true)] out T? whiteEntry) where T : WhiteEntry, new()
```

Comment:
below is:

```
if (EndPointParser.TryParse(parts.LastOrDefault(), network.DefaultPort, out EndPoint endPoint))
```

which can be rewritten to:

```
if (EndPointParser.TryParse(parts.LastOrDefault(), network.DefaultPort, out EndPoint? endPoint))
```

Reply:
Done